<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBSubdivisionDlg</class>
 <widget class="QDialog" name="GBSubdivisionDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GBSubdivisionDlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Input iImages</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="input_imgs_txt">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="input_btn">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Cut method</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="cut_method_box">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Standard frame cutting</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Graph symbol cutting</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="scale_label">
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="scale_box">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>1:500</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1:1000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1:2000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1:5000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1:10000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1:25000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1:50000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1:100000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1:250000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1:500000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1:1000000</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="graph_sym_label">
     <property name="text">
      <string>Graph Symbol</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="graph_sym_txt">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QCheckBox" name="use_graph_check">
     <property name="text">
      <string>Using all graph symbol</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Graph Symbol</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Output path</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="output_path_txt">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="output_path_btn">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Filename prefix</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="file_pre_txt">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Filename suffix</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="file_suf_txt">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Output format</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QComboBox" name="out_format_box">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Expansion method</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QComboBox" name="expand_method_box">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Rectangle expansion</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Polygon expansion</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Resampling method</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QComboBox" name="resample_method_box">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>NEAREST</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BILINEAR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CUBIC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GAUSS</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>No data value</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QLineEdit" name="no_data_txt">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Expansion pixel</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QLineEdit" name="expand_pixel_txt">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Output resolution</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>x:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="x_res_txt">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="y_res_txt">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="15" column="2">
    <widget class="QCheckBox" name="keep_out_check">
     <property name="text">
      <string>Keep out of the image</string>
     </property>
    </widget>
   </item>
   <item row="16" column="2">
    <widget class="QCheckBox" name="out_check">
     <property name="text">
      <string>Output cutting edge</string>
     </property>
    </widget>
   </item>
   <item row="17" column="2">
    <widget class="QCheckBox" name="build_overview_check">
     <property name="text">
      <string>Building overview</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_btn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="2">
    <widget class="QTableWidget" name="graph_tab">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Sign</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Symbol</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
